libcontainer/criu_opts.go

```diff
@@ -1,5 +1,14 @@
 package libcontainer
 
+type cg_mode uint32
+
+const (
+	CRIU_CG_MODE_SOFT cg_mode = 3 + iota
+	CRIU_CG_MODE_FULL
+	CRIU_CG_MODE_STRICT
+	CRIU_CG_MODE_DEFAULT
+)
```

Comment:
@avagin Constants are defined here according to Ruslan's criu patch. The number begins with 3; as restoring a container requires cgroup. Thanks.